api-responses/make_api_responses.sh

```diff
@@ -0,0 +1,66 @@
+# The following script sets up two test nodes,
+# Test node 1: 2 OpenNeuro datasets, in the aggregate results mode
+# Test node 2: the (default) BIDS Synthetic dataset, in the unaggregated results mode
+#
+# A full_stack deployment recipe will be used for Test node 1 to provide local federation.
+
+mkdir test_node1 test_node2 api-responses
+
+# SETUP FOR TEST NODE 1
+git clone https://github.com/neurobagel/recipes.git test_node1/recipes
+
+# Get the data
+git clone https://github.com/neurobagel/openneuro-annotations.git test_node1/openneuro-annotations
+mkdir test_node1/openneuro_mini_data
+cp test_node1/openneuro-annotations/ds000001.jsonld test_node1/recipes/ds000001.jsonld test_node1/openneuro_mini_data
```

Comment:
Is that intentionally the same dataset twice?

Reply:
Ah, that was a typo - thanks for catching that